ui/app/src/views/HomeDashboard.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Box, useTheme } from '@mui/material';
+import {
+  ErrorAlert,
+  ChartsThemeProvider,
+  generateChartsTheme,
+  EChartsTheme,
+  PersesChartsTheme,
+} from '@perses-dev/components';
+import { PluginRegistry, PluginBoundary } from '@perses-dev/plugin-system';
+import { DashboardResource } from '@perses-dev/core';
+import ViewDashboard from '../views/ViewDashboard';
+import { DataSourceRegistry } from '../context/DataSourceRegistry';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { useBundledPlugins } from '../model/bundled-plugins';
+import { useSampleData } from '../utils/temp-sample-data';
+
+// app specific echarts option overrides, empty since perses uses default
+// https://apache.github.io/echarts-handbook/en/concepts/style/#theme
+const ECHARTS_THEME_OVERRIDES: EChartsTheme = {};
+
+const DEFAULT_DASHBOARD_ID = 'node-exporter-full';
+
+function HomeDashboard() {
+  const { getInstalledPlugins, importPluginModule } = useBundledPlugins();
+
+  const [searchParams] = useSearchParams();
+
+  // TODO: completely remove old dashboard url param approach
+  const dashboardParam = searchParams.get('dashboard');
+  const dashboard = useSampleData<DashboardResource>(dashboardParam || DEFAULT_DASHBOARD_ID);
+
+  const muiTheme = useTheme();
+  const chartsTheme: PersesChartsTheme = useMemo(() => {
+    return generateChartsTheme('perses', muiTheme, ECHARTS_THEME_OVERRIDES);
+  }, [muiTheme]);
+
+  // TODO: Loading indicator
+  if (dashboard === undefined) {
+    return null;
+  }
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100vh',
+      }}
+    >
+      <Header />
+      <Box
+        component="main"
+        sx={{
+          flexGrow: 1,
+          overflow: 'hidden',
+        }}
+      >
+        <ChartsThemeProvider themeName="perses" chartsTheme={chartsTheme}>
+          <PluginRegistry getInstalledPlugins={getInstalledPlugins} importPluginModule={importPluginModule}>
+            <PluginBoundary loadingFallback="Loading..." ErrorFallbackComponent={ErrorAlert}>
+              <DataSourceRegistry>
+                <ViewDashboard dashboardResource={dashboard} />
```

Comment:
So I separated them since I wanted to ensure dashboardResource was defined since spec.duration is needed when determining the defaultTimeRange for TimeRangeStateProvider. That part will change though when we remove the useSampleData approach and connect BE & FE.

I'm [still confused ](https://github.com/perses/perses/pull/530#discussion_r971240326)with how things are named in ui/dashboards though, didn't we have a view rendering another view before my PR? Might need a call to talk this through 😄

Reply:
Maybe something like [this commit ](https://github.com/perses/perses/commit/94ba4885a02423906de500d10577e3fe1aa337d7#diff-c8d994442c02c8a3708b3011ec892bc3b5514bf9ba2150e965b179d0490c7a37) could work, ended up renaming DashboardApp in ui/dashboards

Reply:
i think it's confusing too, we have too many "views" 😆

Reply:
Oh, I think I'm starting to understand. So I would create a folder for the overall DashboardView (e.g. `DashboardView` or whatever) and then the main view component is the export from that folder (probably in an `index.ts` file). Then if you need sub-components here at the view level, that's totally cool, you can put them in the folder next to the main view. Same pattern we use for components that have sub-components in separate files basically. We do this for view-specific stuff in our internal codebase too.

Reply:
See comment [here](https://github.com/perses/perses/pull/530#discussion_r975450689), I ended up not needing to do the reorg as part of this PR, but I think we should look into simplifying those view components later.